src/Forms/Login.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './Login.css';
+import { useDispatch } from 'react-redux';
+import { loginUser } from '../redux/users/usersSlice';
+import { baseURL } from '../helpers/api';
+
+const Login = () => {
```

Comment:
- [ ] For some reason, I can't login or access the API at all as a new user with no localStorage:

![image](https://user-images.githubusercontent.com/57421823/223740686-301ebf92-1534-427f-b7c9-ad6f9ccba480.png)

![image](https://user-images.githubusercontent.com/57421823/223740831-8d838f5d-b852-4cdf-a8ff-dbc2f56aa524.png)
****

Also, the live demo [API](https://i-model-cars.onrender.com/api/v1) on your readme gives me a blank page.  Is the server running and deployed?  I can see the frontend app points directly to this deployed app.

Also, the frontend live demo, seems to have the same problem.

## Chrome console:

![image](https://user-images.githubusercontent.com/57421823/223744862-1a172563-bd79-4498-aac7-5211d594cdc1.png)

![image](https://user-images.githubusercontent.com/57421823/223744943-5691d0d1-bf88-4e1d-872b-2d5a951f37a3.png)

## Firefox console

![image](https://user-images.githubusercontent.com/57421823/223745122-470cd940-fdec-4e11-974a-9ee8327c1bb5.png)

![image](https://user-images.githubusercontent.com/57421823/223745426-fef74bb2-eb16-444f-bca7-7244650717a0.png)



Tried deleting localStorage and cookies.  Firefox indicates a problem with [cors](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin)

Tested both in Chrome and Firefox

To replicate the issue, please delete your localStorage and try to access as a new user.

I couldn't access the app.  Kindly fix this bug so your app works as intended.   Otherwise, if there are special instructions to run your app, kindly include then in your readme.👍

Reply:
if the problem persists please change the value of `baseURL` variable in `helpers/api.js` to `"localhost:3000/api/v1"` and then run the backend server locally before starting the react app.